tests/flow/index_utils.py

```
@@ -0,0 +1,143 @@
+    if idx_type == "RANGE":
+        idx_type = ""
+
+    q = f"CREATE {idx_type} INDEX FOR {pattern} ON ("
+    q += ",".join(map("e.{0}".format, properties))
+    q += ")"
+
+    if options is not None:
+        # convert options to a Cypher map
+        options_map = "{"
+        for key, value in options.items():
+            if type(value) == str:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)